UI/src/app/components/comfy/comfy.component.html

```diff
@@ -78,8 +78,22 @@ <h4>Source tables
     </article>
     <article>
       <h4>Target tables
-        <ng-container *ngIf="data.target?.length">({{data.target.length}})</ng-container>
+        <mat-label style = "padding-right: 15px;" *ngIf="data.target?.length">({{data.target.length}})</mat-label>
+        <mat-label 
+        style = "padding-left: 15px; border-left: 2px solid black; color: gray;" 
+        *ngIf="cdmFilter?.selectedTypes.length"
+        [matTooltip]="cdmFilter.selectedTypes.join(', ')"
+        matTooltipPosition="above"
+        >
+        {{cdmFilter.selectedTypes.join(', ').length>25 ? 
+        (cdmFilter.selectedTypes.join(', ') | slice:0:25) + '..':
```

Comment:
You don't need any logic here. You need only css for that(text-overflow: ellipsis;)
https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow